build/configs/rtl8730e/cxxtest/defconfig

```diff
@@ -1377,6 +1392,7 @@ CONFIG_SYSTEM_NETDB_STACKSIZE=2048
 CONFIG_SYSTEM_NETDB_PRIORITY=100
 # CONFIG_SYSTEM_RAMTEST is not set
 # CONFIG_SYSTEM_READLINE is not set
+# CONFIG_READLINE_ECHO is not set
```

Comment:
I think you don't modify this using menuconfig. Because when `CONFIG_SYSTEM_READLINE` is disabled, this line should not be added.

Reply:
Sorry for this, I have tried to align the defconfigs with menuconfig.
If anything else is incorrect, please let me know and I will fix it.